pepper-apis/studybuilder-cli/studies/singular/patches/ddp-8554-parental-validations.conf

```diff
@@ -0,0 +1,84 @@
+{
+  "validations": [
+
+    # When using the Enroll Child flow, if a user enters an age between >=7 years and AOM for their region
+    # and answers NO to cognitive-impairment in the pre-qual but enters a date of birth that equates to <7 years
+    # in the child consent, an error message should appear under the consent DOB field
+    # Formally: PREQUAL.AGE > 7 && !PREQUAL.CAPABLE && CONSENT.AGE <= 7
+    {
+      "messageTemplate": {
+        "templateType": "HTML",
+        "templateText": "$singular_consent_parental_validation_case1",
+        "variables": [
+          {
+            "name": "singular_consent_parental_validation_case1",
+            "translations": [
+              {
+                "language": "en",
+                "text": """Please check the date of birth you entered for your child. If the date is correct,
+                               then your child does not need to give their own assent to participate in the study.
+                               In that case, please return to the dashboard and click "Enroll my child" to begin a
+                               new enrollment for your child, or contact us for help at (650) 561-6750 or
+                               <a href="mailto:contact@projectsingular.org">contact@projectsingular.org</a>.
+                            """
+              }
+            ]
+          }
+        ]
+      },
+      "stableIds": ["CONSENT_PARENTAL_CHILD_DATE_OF_BIRTH", "ENROLLING_CHILD_AGE, ADD_PARTICIPANT_INCAPACITATED"],
+      "precondition": """
+                          user.studies["singular"].forms["ADD_PARTICIPANT_PARENTAL"].questions["ENROLLING_CHILD_AGE"].isAnswered()
+                          &&
+                          user.studies["singular"].forms["ADD_PARTICIPANT_PARENTAL"].questions["ADD_PARTICIPANT_INCAPACITATED"].isAnswered()
+                          &&
+                          user.studies["singular"].forms["CONSENT_PARENTAL"].questions["CONSENT_PARENTAL_CHILD_DATE_OF_BIRTH"].isAnswered()
+                          """,
+      "expression": """
+                        user.studies["singular"].forms["ADD_PARTICIPANT_PARENTAL"].questions["ENROLLING_CHILD_AGE"].answers.value() > 7
+                        &&
+                        user.studies["singular"].forms["ADD_PARTICIPANT_PARENTAL"].questions["ADD_PARTICIPANT_INCAPACITATED"].answers.hasFalse()
+                        &&
+                        user.studies["singular"].forms["CONSENT_PARENTAL"].questions["CONSENT_PARENTAL_CHILD_DATE_OF_BIRTH"].answers.ageAtMost(7, YEARS)
+                        """
+    },
+
+    # When using the Enroll Child flow, if a user enters an age between >=7 and AOM for their region and
+    # answers NO to cognitive-impairment in the pre-qual but enters a date of birth that equates to >=AOM
+    # in the child consent, an error message should appear under the consent DOB field.
+    # Formally: PREQUAL.AGE < 7 && CONSENT.AGE >= 7
```

Comment:
Once again its AGE entered in ADD_PARTICIPANT_PARENTAL not PREQUAL

Reply:
I don't use PREQUAL anywhere. I just used the same notation in the comments that Whitney did.